frontend/elements/src/entry/jig/publish/resources.ts

```diff
@@ -0,0 +1,31 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import "@elements/core/titles/variants/title-w-icon";
+@customElement('resources-column')
+export class _ extends LitElement {
+  static get styles() {
+    return [css`
+
+    
+    `];
+  }
+
+
+  @property({type:Boolean})
+  uploaded: boolean = false;
+  
+
+  render() {
+
+    const {uploaded} = this;
+    
+    return html`
+    <div>
+        <title-wicon title="Add lesson plan" path="Icn_Add.svg"  .uploaded=${uploaded}></title-wicon>
```

Comment:
the element name should be title-w-icon to match the typescript filename... though this will apply to lots of areas. I suppose leave it for now and resolve this comment just to make sure you've seen it, and we can open a new issue later

Reply:
Fixed

Reply:
the element names are still `title-wicon` here

Reply:
was fixed just missed to commit